sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -289,19 +311,25 @@ def _load_raw_query_metrics_rows(self, cursor):
 
     def _normalize_queries(self, rows):
         normalized_rows = []
+
         for row in rows:
+            is_proc = row['object_name'] is not None
```

Comment:
Do we have concern that datadog user is able to get the `object_id` of the stored procedure from `sys.dm_exec_procedure_stats` but do not have the permission to resolve the object name? According to [doc](https://learn.microsoft.com/en-us/sql/t-sql/functions/object-name-transact-sql?view=sql-server-ver16#exceptions), lacking permission could return object name NULL.

> A user can only view the metadata of securables that the user owns or on which the user has been granted permission. This means that metadata-emitting, built-in functions such as OBJECT_NAME may return NULL if the user does not have any permission on the object.